docs/luigi-core-api.md

```diff
@@ -1321,6 +1322,46 @@ Returns **[Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Gl
 
 -   **since**: 1.4.0
 
+#### isValid
```

Comment:
I believe **isValid** and is **isDuplicatedOrDisabled** are not intended to be publicly available API calls as they are internal class helper functions, unless it was specifically required . 
They are most likely automatically added here because they are in the core-api/ folder. I think you can add the following comment similar to [here](https://github.com/SAP/luigi/blob/master/core/src/core-api/config.js#L177) and then run **npm run docu** again to remove the docu about these functions:

`   * @private`

Reply:
Good catch! Thanks for pointing this out.